bindings/Makefile

```diff
@@ -16,11 +17,28 @@ deps/$(CODEGEN_BIN):
 build/%: $(SWAGGER_SPEC) deps/$(CODEGEN_BIN)
```

Comment:
Since this no longer matches multiple targets, let's just make this `build/typescript-fetch` rather than `build/%`.  You'll also have to change the `$*`'s.